hubploy/auth.py

```diff
@@ -118,6 +133,20 @@ def cluster_auth_gcloud(deployment, project, cluster, zone, service_key):
     )
     subprocess.check_call(gcloud_cluster_credential_command)
 
+    yield current_login
+
+
+@contextmanager
```

Comment:
i don't believe that i need to use this decorator...  but i'm unsure and not an expert in this bit of python.  :)